scripts/obtain/dotnet-install.sh

```diff
@@ -146,8 +150,8 @@ get_current_os_name() {
         local linux_platform_name
         linux_platform_name="$(get_linux_platform_name)" || { echo "linux" && return 0 ; }
 
-        if [[ $linux_platform_name == "rhel.6"* ]]; then
-            echo "rhel.6"
+        if [[ $linux_platform_name == "rhel.6" || $linux_platform_name == "alpine.3.6" ]]; then
```

Comment:
Did you remove a `*` here on purpose?  Did that change this check?

Reply:
I have removed it since it is not needed. We always generate rhel.6 for RHEL / CentOS 6 in the code above.